doc/users/dflt_style_changes.rst

```diff
@@ -405,6 +405,73 @@ or by setting ::
 in your :file:`matplotlibrc` file.
 
 
+``boxplot``
+-----------
+
+Previously, boxplots were composed of a mish-mash styles that were, for
+better for worse, inherited from Matlab. Most of the elements were blue,
+but the medians were red. The fliers (outliers) were black plus-symbols
+(`+`) and the whiskers were dashed lines, which created ambiguity if
+the (solid and black) caps were not drawn.
+
+For the new defaults, everything is black except for the median and mean
+lines (if drawn), which are set to the first two elements of the current
+color cycle. Also, the default flier markers are now hollow circles,
+which maintain the ability of the plus-symbols to overlap without
+obscuring data too much.
+
+.. plot::
+
+    fig, (old, new) = pyplot.subplots(ncols=2, sharey=True)
```

Comment:
These code blocks are run as part of the doc-build process to generate the example images.